File: src/main/java/org/taktik/connector/technical/validator/SessionValidator.java
package org.taktik.connector.technical.validator;

import org.taktik.connector.technical.exception.SessionManagementException;
import org.taktik.connector.technical.service.sts.security.SAMLToken;

public interface SessionValidator {
   boolean validateSession() throws SessionManagementException;

   boolean validateToken(SAMLToken var1) throws SessionManagementException;
}

File: src/main/kotlin/org/taktik/connector/technical/validator/SessionValidator.kt
package org.taktik.connector.technical.validator

import org.taktik.connector.technical.exception.SessionManagementException
import org.taktik.connector.technical.service.sts.security.SAMLToken

interface SessionValidator {
    @Throws(SessionManagementException::class)
    fun validateSession(): Boolean

    @Throws(SessionManagementException::class)
    fun validateToken(var1: SAMLToken?): Boolean
}
